### PR TITLE
Nadav/assad

### PR DESCRIPTION
**Pull request by monadav**
1904335188
1904335169

### CODE REVIEW

File: a.txt [hunk 1]
```diff
@@ -1 +1,7 @@
-abcd
\ No newline at end of file
+abcd
```
Comment (monadav) [inline a.txt:1]: asd

Comment (monadav) [reply a.txt:1]: asd

**Review by monadav — COMMENTED** [review]
asd

File: a.txt (hunk 1 above)
Comment (monadav) [reply a.txt:1]: qwe

Comment (nadavgld) [reply a.txt:1]: asd2

**Pull request closed without merging**